docs/getting_started/user-guide.md

```diff
@@ -489,6 +489,28 @@ def push(uri, root):
 
 </details>
 
+<details>
+
+<summary>Example of basic artifact attachment</summary>
```

Comment:
This looks to be redundant with the other PR - let's do that one first, merge and rebase here, and then we can do the review.

Reply:
Yeah, I know... that's why I left this PR as a draft for now. 🙂  It builds on top of the other PR. I wanted to share the direction with you but wanted to keep PRs atomic at the same time. 🙂 Let's finish the first PR first and I'll rebase on top of it.

Reply:
Sounds good! Thank you for your work on both, and looking forward to coming back to this one.